src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -10,7 +12,11 @@ public class AuthenticationService {
      * @return true if user by email exists and passed password is equal to user's password.
      * Return false in any other cases.
      */
+    private UserService userService = new UserService();
+
     public boolean login(String email, String password) {
-        return false;
+        User user = userService.findByEmail(email);
+        return (userService.findByEmail(email) != null && password.equals(
+                user.getPassword()));
```

Comment:
```suggestion
        return user  != null && password.equals(user.getPassword());
```

Reply:
@Rommelua Please, check if I have any errors in code and the program works correct. Thank you so much !